### ISSUE TITLE
Work on a DEV branch. 

### ISSUE THREAD

**Issue by Jim8y** [neo-project/neo#3368]
**Summary or problem description**
I think we should work on a dev branch, instead of on the master branch directly. @vncoelho has mentioned that he found some issues in prs that were reviewed and merged into the master branch. Though i could not agree with some of his discoveries, I agree with his opinion that we should not directly merge pr into master without a deep test. However, running a deep test on every pr is not practical.

**Do you have any solution you want to propose?**
**i suggest that we work on a DEV branch, then we open a pr to merge that dev branch into the master every month, and do a deep test with that pr.**

**Where in the software does this update applies to?**
- Github

**Comment by shargon** [comment]
Duplicated with https://github.com/neo-project/neo/issues/3270 ?

**Comment by roman-khimov** [comment]
Exactly. The older one wins in this case, so I'm closing this duplicate.
